### DISCUSSION TITLE
Getting the wrong event ref in github

### DISCUSSION THREAD

**Discussion by psilore** [renovatebot/renovate#27663] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

Why am I getting github event ref `branches/rv8-patch-github-actions` when I am running a workflow that triggers on delete?

![image](https://github.com/renovatebot/renovate/assets/366256/ea5a527a-54e9-49f4-8299-2e64aa008d83)

There is no trace of the ref that starts with `branches/rv8-patch-github-actions` in the logs

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
I'm not sure how this relates to Renovate

**Reply by psilore** [reply]
The workflow trigger whenever that type of branch `branches/rv8-patch-github-actions` gets deleted.
and it produces errors due to that the github event ref is containing `branches/rv8-patch-github-actions` and not `refs/heads/rv8-patch-github-actions`

**Reply by viceice** [reply]
It's your workflow trigger which causes that, not renovate.

**Reply by psilore** [reply]
Hmm, maybe. But why does github add the "branches/" to the ref just for renovate branches? (That does not makes sense)

Test:
Last commit id in renovate PR contains "branches/" in ref
https://github.com/psilore/test-delete-event-in-github-with-renovate/actions

Anyway, sorry for bothering you and take up your time. I cant figure it out....its weird
